### ISSUE TITLE
Watch Mode (BrowserSync) not working for both Sage and Laravel

### ISSUE THREAD

**Issue by distributev** [distributev/wordpress-laravel-docker-local-cloud#47]
![image](https://user-images.githubusercontent.com/19224635/36889950-6dae64ee-1dfc-11e8-97e9-4b75c4d68e90.png)

![image](https://user-images.githubusercontent.com/19224635/36899843-2c762492-1e21-11e8-8a3b-921c7e11f5ed.png)

**Comment by distributev** [comment]
![lara](https://user-images.githubusercontent.com/19224635/36911304-6bd2fc7e-1e43-11e8-91a3-ceeccec1e996.png)

**Comment by distributev** [comment]
![image](https://user-images.githubusercontent.com/19224635/36911620-5ba9bc38-1e44-11e8-80ba-010e2fbfb9a5.png)

**Comment by distributev** [comment]
first, I figure out why yesterday you had a redirection to https in chrome, it's not because of cache, it's an internal behavior of chrome with .dev tld domain. learn more here https://stackoverflow.com/questions/47768289/chrome-redirects-dev-to-https . 

I changed the default domain from "app.dev" to "app.devel" https://github.com/distributev/wordpress-laravel-docker-local-cloud/commit/275b87a34fc6caa5a540c2a18b36af16a7d33673

Regarding the watch issue, I had the same issue as you the watch is not working, this is not a bug or misconfiguration. 

The watch doesn't work when sharing volume with NFS (which is the case for Docker native) and many people encounter this problem. So we need to enable the poll watching to work around this. I spent a lot of time to figure out how to do it since it's not mentioned in the doc/github/roots forum.

Look at my answer here https://discourse.roots.io/t/browsersync-not-watching-changes-in-docker-for-windows/11275 to see how to enable poll watching for sage. this config is needed only for users in windows
